rabbitmq/requirements-dev.txt

```diff
@@ -1,2 +1,2 @@
 -e ../datadog_checks_dev
-pika==0.10.0
+pika==0.13.0
```

Comment:
Why the bump?

Reply:
It used to use `async` as a variable but that is now a keyword